docs/.vuepress/config.ts

```diff
@@ -5,6 +5,10 @@ import theme from "./theme.js";
 import { pwaPlugin } from "vuepress-plugin-pwa2";
 import { getDirname, path } from "@vuepress/utils";
 
+// import registerPkg from "@vuepress/plugin-register-components";
```

Comment:
Can we remove these comments please?

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/657/commits/fef65a21bb07e38ff0d91c7b2fda556f3ea50ae9